steam/tag.py

```diff
@@ -35,6 +35,21 @@ class StoreCategory(PartialCategory):
     image: CDNAsset
 
 
+@dataclass(slots=True)
+class PartialGenre:
+    _state: ConnectionState
+    id: int
+
+    # TODO
+    async def fetch(self) -> StoreGenre:
+        ...
```

Comment:
I'm curious if there's a url where I can retrieve this for genres and categories?

Reply:
When I'm unsure about these things my first port of call is https://github.com/Revadike/InternalSteamWebAPI/wiki. https://github.com/Revadike/InternalSteamWebAPI/wiki/Get-Categories-By-Tag seems like a bad way to do this but it could work for categories if you filter from it (realistically this should be its own method e.g. `Client.all_categories()`). I'll have look through the store page for something better

Reply:
Ooh there are some really cool endpoints here, thanks